### PR TITLE
BACKLOG-15637 add-in the C3218086 C3218088 C3218089 test-case with additional functionalities

### PR DESCRIPTION
**Pull request by XiangYuHe**
## JIRA

https://jira.jahia.org/browse/BACKLOG-15637

## Description

A bit of re-factor plus add in the test cases for C3128086 per the test rail

Updated per the latest PR to add C3218088 and C3218089 as well in this PR
Simplified the scenarios for both so don't leverage the "digitall" project at all.

### CODE REVIEW

**Comment by VladRadan** [comment]
@XiangYuHe why did you create those `.json` files in fixtures instead of just having those as variable in the spec?

File: tests/cypress/integration/mysite.home.spec.ts [hunk 1]
```diff
@@ -57,4 +62,118 @@ describe('Enable sitemap on MySite', () => {
...
+        // Step #1 the standard enable
+        siteHomePage
+            .goTo(siteUrlHome)
+            .editPage('My Site')
+            .clickOnSitemap()
+            .clickOnSave()
+            .validateSucessMessage()
+            .clickBack()
+        // Step #2 publishing the site and flush the cache
+        siteHomePage.publishSite('My Site').clickPublishAll().flushCache()
```
Comment (sdusolle) [inline tests/cypress/integration/mysite.home.spec.ts:140]: I think we are missing a step here, in Testrail we have :
_- Open page composer and edit page /home/our-companies
- Open Live Roles and remove Guest & users from readers
- Add bill as user
- Save_

Comment (XiangYuHe) [reply tests/cypress/integration/mysite.home.spec.ts:140]: I simplified the portion here
As I don't see any reason we need to add the user and save
By going through the SEO portion is enough for testing the no-index

**Comment by XiangYuHe** [comment]
> @XiangYuHe why did you create those `.json` files in fixtures instead of just having those as variable in the spec?

As fixture is used as static test data
I feel it is better to put results there (makes more sense for me)

File: tests/cypress/support/commands.ts [hunk 2]
```diff
@@ -41,6 +41,7 @@ Cypress.Commands.add('requestFindNodeInnerHTMLByName', function (url: string, no
             .find(nodeName)
             .toArray()
             .map((el) => el.innerText)
+        JSON.stringify(nodes)
```
Comment (AKarmanov) [inline tests/cypress/support/commands.ts:44]: This doesn't seem to be doing anything as return value not used

**Review by AKarmanov — APPROVED** [review]

File: tests/cypress/support/commands.ts (hunk 2 above)
Comment (XiangYuHe) [reply tests/cypress/support/commands.ts:44]: You are right I will remove this (forgot to remove)

**Pull request merged**